py3-SecretStorage.yaml

```diff
@@ -0,0 +1,46 @@
+# Generated from https://pypi.org/project/SecretStorage/
+package:
+  name: py3-SecretStorage
+  version: 3.3.3
+  epoch: 0
+  description: Python bindings to FreeDesktop.org Secret Service API
+  copyright:
+    - license: BSD 3-Clause License
+  dependencies:
+    runtime:
+      - py3-cryptography
+      - py3-jeepney
+      - python3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+      - py3-build
+      - py3-installer
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 3993fc7162fe50852852913b90b203ba13727c38
+      repository: https://github.com/mitya57/secretstorage
+      tag: ${{package.version}}
+
+  - name: Python Build
+    runs: |
```

Comment:
Should this and all the others use:
python/build-wheel pipeline instead?

Reply:
Oh fancy! I didn't know we had that one yet.